src/SprykerMiddleware/Zed/Process/Dependency/Plugin/TranslatorStagePluginInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace SprykerMiddleware\Zed\Process\Dependency\Plugin;
+
+use Generated\Shared\Transfer\TranslatorConfigTransfer;
+
+interface TranslatorStagePluginInterface extends StagePluginInterface
```

Comment:
Usually, it's better to follow interface segregation principle instead of inheritance (eg. TranslationAwareStagePluginInterface). Does it fit for you here?